props/saber_fett263_buttons.h

```diff
@@ -293,248 +659,3857 @@ class SaberFett263Buttons : public virtual PropBase {
+        default:
+        case 2:
+          sound_queue_.Play(SoundToPlay(&SFX_mnum, (tens * 10) - 1));
+          break;
+        case 3:
+          sound_queue_.Play(SoundToPlay("thirty.wav"));
+          break;
+        case 4:
+          sound_queue_.Play(SoundToPlay("forty.wav"));
+          break;
+        case 5:
+          sound_queue_.Play(SoundToPlay("fifty.wav"));
+          break;              
+        case 6:
+          sound_queue_.Play(SoundToPlay("sixty.wav"));
+          break;  
+        case 7:
+          sound_queue_.Play(SoundToPlay("seventy.wav"));
+          break;
+        case 8:
+          sound_queue_.Play(SoundToPlay("eighty.wav"));
+          break;  
+        case 9:
+          sound_queue_.Play(SoundToPlay("ninety.wav"));
+          break;  
+      }
+      if (number != 0) sound_queue_.Play(SoundToPlay(&SFX_mnum, number - 1));
+    }
+  }
+
+  void SayNumber (float number, SayType say_type) {
+    int thousand = ((int)floorf(number / 1000)) % 100;
+    int hundred = ((int)floorf(number / 100)) % 10;
+    int ones = ((int)floorf(number)) % 100;
+    int tenths = ((int)floorf(number * 10)) % 10;
+    int hundredths = ((int)floorf(number * 100)) % 10;
+    switch (say_type) {
+      case SAY_BATTERY:
+      case SAY_DECIMAL:
+        // Battery Level prompt
+        if (say_type == SAY_BATTERY) sound_queue_.Play(SoundToPlay("mbatt.wav"));
+        // Tens & Ones
+        if (number == 0) {
+          sound_queue_.Play(SoundToPlay("mzero.wav"));
+        } else {
+          TensValue(ones);
+        }
+        // Decimal / Point
+        sound_queue_.Play(SoundToPlay("mpoint.wav"));
+        // Tenths
+        if (tenths == 0) {
+          sound_queue_.Play(SoundToPlay("mzero.wav"));
+        } else {
+          sound_queue_.Play(SoundToPlay(&SFX_mnum, tenths - 1));
+        }
+        // Hundredths
+        if (hundredths != 0) sound_queue_.Play(SoundToPlay(&SFX_mnum, hundredths - 1));
+        // Volts prompt
+        if (say_type == SAY_BATTERY) sound_queue_.Play(SoundToPlay("mvolts.wav"));
+        break;
+      case SAY_WHOLE:
+        // Thousands
+        if (thousand > 0) {
+          TensValue(thousand);
+          sound_queue_.Play(SoundToPlay("thousand.wav"));
+        }
+        // Hundred
+        if (hundred > 0) {
+          sound_queue_.Play(SoundToPlay(&SFX_mnum, hundred - 1));
+          sound_queue_.Play(SoundToPlay("hundred.wav"));          
+        }
+        // Tens & Ones
+        if (ones == 0) {
+          if ((thousand + hundred) == 0) sound_queue_.Play(SoundToPlay("mzero.wav"));
+        } else {
+          TensValue(ones);
+        }
+        break;
+      default:
+        break;
+    }
+  }
+   
+  enum ClashType {
+    CLASH_NONE,
+    CLASH_CHECK,
+    CLASH_BATTLE_MODE,
+    CLASH_LOCKUP,
+    CLASH_LOCKUP_END,
+#ifdef FETT263_CLASH_STRENGTH_SOUND    
+    CLASH_NORMAL,
+    CLASH_STAB,
+#endif
+  };
+
+#ifdef FETT263_CLASH_STRENGTH_SOUND
+  void RealClashSound() {
```

Comment:
I think this an weird name.
Real clash sound? As opposed to what?
Maybe HandleClash() ?

Reply:
renamed to HandleClash()